blog.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>BINABOX- The Basilikis NFT and Crytpt Gaming Insights</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <header>
```

Comment:
header and footer are different on blog and index page, it's better to copy one version to both pages